example_app/web/templates/layout/app.html.eex

```diff
@@ -25,7 +27,5 @@
       </main>
 
     </div> <!-- /container -->
-    <script src="<%= static_path(@conn, "/js/jquery.min.js") %>"></script>
```

Comment:
I know it's a best practice to have the JS load at the bottom of the page, but it was easiest to move it to the `<head>` to be able to put in some inline JS to bind the button in the template.  

Alternatively I could have put the button binding in app.js or a template-specific JS file. I'm down for whichever on this, not sure how much we care about JS cleanliness in this example app